src/renderer/app.vue

```
@@ -83,4 +94,13 @@
     };
   },
 });
+
+//
+
+const error1 = {
```

ESLint

[error] 'error1' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)